README.md

```diff
@@ -292,6 +292,40 @@ const indexPost = async ({ text, id, title, published, author }) => {
 };
 ```
 
+If `MOCK_ELASTIC=1` is set in the environment variables, the `Elastic()` function creates a `mock` object and creates an instance of [ElasticSearch-mock](https://www.npmjs.com/package/@elastic/elasticsearch-mock) for testing.
+
+```js
+const { mock } = Elastic();
```

Comment:
This is correct, but since you're going to use it below, why not use the `client` too?

```js
const client = Elastic();
const mock = client.mock;
```

Reply:
So, I'm looking at the mocks here https://github.com/elastic/elasticsearch-js-mock/blob/main/test.js, they seem to have one mock and one client for each test?
So, if we do this way with
```
const client = Elastic();
const mock = client.mock;
```
Then we probably don't need to have `mock.clearAll()` before each test?

Reply:
Except that we're re-using that `mock` for every test, so you might want to have it be reset so you don't have old mock routes added?

Up to you, both ways work.

Reply:
In Telescope we'd have to use it with 
```js
const { mock } = Elastic();
```
Because the client is in `search.js`

Reply:
Right.  The docs here are just showing other ways of accessing it so that it's clear.